checkov/kustomize/runner.py

```diff
@@ -674,47 +671,21 @@ def run_kustomize_to_k8s(
             if self.kustomizeProcessedFolderAndMeta[file_path].get('type') == 'overlay':
                 self._handle_overlay_case(file_path)
 
-        if platform.system() == 'Windows':
-            if not self.templateRendererCommand:
-                logging.error("The 'templateRendererCommand' was not set correctly")
-                return
-
-            shared_kustomize_file_mappings: dict[str, str] = {}
-            for file_path in self.kustomizeProcessedFolderAndMeta:
-                self._run_kustomize_parser(
-                    file_path=file_path,
-                    shared_kustomize_file_mappings=shared_kustomize_file_mappings,
-                    kustomize_processed_folder_and_meta=self.kustomizeProcessedFolderAndMeta,
-                    template_renderer_command=self.templateRendererCommand,
-                    target_folder_path=self.target_folder_path,
-                )
-            self.kustomizeFileMappings = shared_kustomize_file_mappings
+        if not self.templateRendererCommand:
+            logging.error("The 'templateRendererCommand' was not set correctly")
             return
 
-        manager = multiprocessing.Manager()
-        # make sure we have new dict
-        shared_kustomize_file_mappings = pickle_deepcopy(manager.dict())  # type:ignore[arg-type]  # works with DictProxy
-        shared_kustomize_file_mappings.clear()
-
-        jobs = []
-        for filePath in self.kustomizeProcessedFolderAndMeta:
-            p = multiprocessing.Process(
-                target=self._run_kustomize_parser,
-                args=(
-                    filePath,
-                    shared_kustomize_file_mappings,
-                    self.kustomizeProcessedFolderAndMeta,
-                    self.templateRendererCommand,
-                    self.target_folder_path
-                )
+        shared_kustomize_file_mappings: dict[str, str] = {}
+        for file_path in self.kustomizeProcessedFolderAndMeta:
```

Comment:
this makes the whole `kustomize` processing sequential, right?

Reply:
This is a temporary solution because, there is a central place for managing parallel processes (parallel_runner.py) and this code is creating processes by itself, which cause issues, so until it will also use parallel_runner I prefer to disable it's feature